src/actions/blockchain.ts

```diff
@@ -765,11 +765,7 @@ export const claimSellerFundsFromSeveral = (
     console.log('withdraw tx logs', withdrawLogs)
 
     // Find the 'NewWithdrawal' log
-    let withdrawEvents
-    // loop until sellBalance drops to 0
-    while (!withdrawEvents) {
-      withdrawEvents = withdrawLogs.find((log: Web3EventLog) => log._eventName === 'NewWithdrawal')
-    }
+    const withdrawEvents = withdrawLogs.find((log: Web3EventLog) => log._eventName === 'NewWithdrawal')
```

Comment:
without this the UI doesnt update with the receipt and the button to claim is still there